app/models/live.rb

```diff
@@ -4,4 +4,8 @@ class Live < ApplicationRecord
   belongs_to :author, class_name: 'User'
 
   has_many :votes
+
+  def can_be_deleted?
+    votes.count.zero?
```

Comment:
I would use `.exists?` instead of `.count.zero?` here, what do you think?

Reply:
Nice one, @raphaeltelatim!
I'd use `votes.any?` 😄

```suggestion
    votes.any?
```

Reply:
`.count` is a bad option indeed, as it hits the database **everytime**!

This thread made me go a bit deeper into this topic and I found out that we **already** have something very nice in place.

Check this out:
https://github.com/lucascaton/easy-lives/blob/826f6ab93c575a17db25e78097526d61e0d9825c/db/migrate/20190419234547_add_votes_count_column_to_lives.rb#L1-L5

We could simply make use of it, like so:

```suggestion
    votes_count.zero?
```

Reply:
@thiagoalessio, This! Totally forgot about counter_cache. Much much better indeed

Reply:
+1